ARCHITECTURE_DECISION_RECORDS/README.md

```diff
@@ -0,0 +1,64 @@
+# Architecture Decision Records
```

Comment:
Seems like a very good start 💪

I would place this README into `docs/adr/` or just `adr/` (to be aligned with the rest of the repo, not a strong opinion).

Reply:
Fair enough. I moved changed to use a top level folder called "ADRs".